src/ResourceParser.h

```diff
@@ -336,6 +334,29 @@ namespace snowcrash {
             }
         }
 
+        /**
+         * \brief Parse the current node as an action
+         */
+        static MarkdownNodeIterator processNestedAction(const MarkdownNodeIterator& node,
+                                  const MarkdownNodes& siblings,
+                                  SectionParserData& pd,
+                                  SectionLayout& layout,
+                                  const ParseResultRef<Resource>& out) {
```

Comment:
The indentation of the above 4 lines should align with the first argument in the function.